[2021-03-09 12:20:26,578] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: my_test_dag.dummy_task 2021-03-08T12:00:00+00:00 [queued]>
[2021-03-09 12:20:26,582] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: my_test_dag.dummy_task 2021-03-08T12:00:00+00:00 [queued]>
[2021-03-09 12:20:26,582] {taskinstance.py:866} INFO - 
--------------------------------------------------------------------------------
[2021-03-09 12:20:26,582] {taskinstance.py:867} INFO - Starting attempt 1 of 1
[2021-03-09 12:20:26,582] {taskinstance.py:868} INFO - 
--------------------------------------------------------------------------------
[2021-03-09 12:20:26,663] {taskinstance.py:887} INFO - Executing <Task(DummyOperator): dummy_task> on 2021-03-08T12:00:00+00:00
[2021-03-09 12:20:26,664] {standard_task_runner.py:53} INFO - Started process 30819 to run task
[2021-03-09 12:20:26,770] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: my_test_dag.dummy_task 2021-03-08T12:00:00+00:00 [running]> rep-pc
[2021-03-09 12:20:26,780] {taskinstance.py:1048} INFO - Marking task as SUCCESS.dag_id=my_test_dag, task_id=dummy_task, execution_date=20210308T120000, start_date=20210309T152026, end_date=20210309T152026
[2021-03-09 12:20:36,507] {logging_mixin.py:112} INFO - [2021-03-09 12:20:36,507] {local_task_job.py:103} INFO - Task exited with return code 0
